fetch_pdftools

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Copyright (c) 2018  Zotero
+#                     Center for History and New Media
+#                     George Mason University, Fairfax, Virginia, USA
+#                     http://zotero.org
+# 
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+DOWNLOAD_URL="https://s3.amazonaws.com/zotero-bucket-example/pdftools.tar.gz"
```

Comment:
Move to config.sh:

```
PDF_TOOLS_VERSION="0.0.1"
PDF_TOOLS_URL="https://zotero-download.s3.amazonaws.com/pdftools/pdftools-$PDF_TOOLS_VERSION.tar.gz"
```

And use `$PDF_TOOLS_VERSION` below (or redirect curl to pdftools.tar.gz instead of using `-O`).

(The version string will make clear which version was in which release.)